src/main/java/net/dries007/tfc/objects/blocks/agriculture/BlockFruitTreeTrunk.java

```diff
@@ -58,7 +58,7 @@ public static BlockFruitTreeTrunk get(IFruitTree tree)
         return MAP.get(tree);
     }
 
-    private final IFruitTree tree;
+    public final IFruitTree tree;
```

Comment:
Hey. Where be my `getFruitTree` with private behind the scenes fields!

Reply:
Fine fine, I also thought on doing it properly, but.... this means i have to do the same thing in 99999 other classes that also has this inconsistency 😨 

(numbers may be a jest, but there are some other blocks that have a public property field that should be private with a getter)

Welp, time to mark this PR as a draft and start refactoring!